File: spec/code_samples/Java/v1@messages@{messageId}/put.java
String messageId = "messageId2";
CancelScheduledMessageRequest body = new CancelScheduledMessageRequest();
body.setStatus("cancelled");

messagesController.cancelScheduledMessageAsync(messageId, body, new APICallBack<DynamicResponse>() {
    public void onSuccess(HttpContext context, DynamicResponse response) {
        // TODO success callback handler
    }
    public void onFailure(HttpContext context, Throwable error) {
        // TODO failure callback handler
    }
});

File: spec/code_samples/JavaScript/v1@messages@{messageId}/put.js
let messageId = 'messageId2';
let body = new lib.CancelScheduledMessageRequest();
body.status = 'cancelled';

const promise = controller.cancelScheduledMessage(messageId, body);
promise.then((response) => {
    // this block will be executed on successful endpoint call
    // `response` will be of type 'mixed'
}, (err) => {
    // this block will be executed on endpoint call failure
    // `err` is an 'object' containing more information about the error
});